images/ironic-rhcos-downloader.yml

```diff
@@ -11,8 +11,7 @@ content:
 enabled_repos:
 - rhel-8-baseos-rpms
 - rhel-8-appstream-rpms
-- rhel-8-server-ose-rpms-embargoed
-- openstack-16-for-rhel-8-rpms
+- rhel-8-server-ironic-rpms
```

Comment:
ironic-rhcos-downloader installs `coreos-installer`, so the OSE repo needs to stay.